src/server/models/users.ts

```diff
@@ -0,0 +1,32 @@
+import { model, Schema, Document, models, Model } from 'mongoose';
+import { User } from '../../types/persons';
+
+const UserSchema = new Schema(
+  {
+    firstName: {
+      type: String,
+      required: false,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+    },
+    donationid: {
+      type: Schema.Types.ObjectId,
+      ref: 'Donation Id',
```

Comment:
This 'ref' field is 'What other collection does this field refer to', so it should be a single word. I believe it should be 'Donation' with what you have set up right now